pkg/controllers/management/aksupstreamrefresh/aks_upstream_refresh.go

```diff
@@ -0,0 +1,180 @@
+package aksupstreamrefresh
```

Comment:
I think it would be ideal to reuse the AKS refresh handler. There may have been some misunderstand when I said if multiple integration handlers need to be made that's fine, since I think at that time it was not understood that the refresh handler is not the same thing as the integration handler. Ideally the cronjob code is all agnostic and depending on which config the custer has custom logic can be ran.

Reply:
I do see that the setting for the refresh cron is called eks-refresh-cron, so that does complicate things. We should discuss the design for this.

Reply:
We agreed on re-implementing `aksupstreamrefresh` into more generic `clusterupstreamrefresh`